### ISSUE TITLE
Moving to resource popup window questions

### ISSUE THREAD

**Issue by JBaltika** [tom-englert/ResXResourceManager#89]
Hi Tom,

The first, thanks for writing & maintaining this fantastic peace of software!  A lot of hard work man ....

I have a few question/enhancements(?):

1. Is possible to remove from key string formatted placeholders for C# moving to resource popup window ? For example : "My name is {0}." or $"My name is {name}."  The result is 'MyNameIs0'

2. Is it possible to remove English 'the' and 'a' as they worthless junk for meaning or readability 

3. Can we have configurable (like drop-down in configuration) how to compose the resource key for example:
{Class}_{Mehtod}_{Key_Phrase}   or
Mehtod}_{Key_Phrase}   or
{Key_Phrase}  just

This would cover most scenarios for most folks. Like we have huge project and we have hundred resx files spitted by module or group so {Key_Phase) its the best, but for other on small projects they might need {Class}_{Mehtod}_{Key_Phrase} which is good practice... 

![shot1](https://user-images.githubusercontent.com/1755340/36856884-2338f88c-1d45-11e8-8875-6a501e9e0732.jpg)

Thanks in advance for your time, Jim

**Comment by tom-englert** [comment]
I found that however hard you try, keys generated from text are usually bad and have to be edited anyhow, so I simply put in as much info as possible, because deleting is easier typing.

However you have some good arguments there, maybe you want to provide a PR with a more sophisticated solution. The code that generates the key is here:

https://github.com/tom-englert/ResXResourceManager/blob/master/ResXManager.VSIX/Visuals/MoveToResourceViewModel.cs#L184

**Comment by JBaltika** [comment]
Hi Tom, 
I think i found solution top please everybody . As developers we overthinking sometimes ... Look that we need just token replacement dictionary essentially.
So user in GUI enters in configuration tab something like key/value pairs
" & "  => " and "
"{0}"  => ""
"The " => ""
" the " => ""  
...
and so on everybody customizes according language and their requirements as pleased

now that what left for you (last part):
```
var builder = new StringBuilder(phrase);
for item in dictionary
builder.Replace(key, value)
```
Very simple no performance overhead like some crazy regex or C# code. Easy implement i guess & pleasing everyone ... What do you think?

**Comment by tom-englert** [comment]
Yes, generally sounds good. However do you think the replace is really necessary?
Replace "&" with "and" and then replace "and" with nothing sounds like overkill. 
If we just provide a list words to remove, and maybe allow regex (there is no need to consider performance, it's executed once per mouse click), configuration would me much simpler.

**Comment by JBaltika** [comment]
Hi Tom,  
This just an example to show that even most picky folks can do what they want for naming  and  your software is great for customization ...  :)
